### ISSUE TITLE
setting safe midle aggressive

### ISSUE THREAD

**Issue by Fungraphic** [Blaksmith/DiceBotScripts#2]
Hi Blaksmith,

Do you have settings for BlaksRunner-4.0.lua :
safe mode
midle mode
aggressive mode
thank you

**Comment by Blaksmith** [comment]
Not really.  The safest mode, would be winMult = 1, with a minimum balance of 1 mil sats (of whichever coin).

The higher you set winMult, the deeper it can dig, but on some good lucky runs, it will gain that much faster.

Changing the base chance lower will help with your higher winMult though too.

**Comment by Fungraphic** [comment]
ok thank you very much for these clarifications

**Comment by Fungraphic** [comment]
![2019-08-01_095908](https://user-images.githubusercontent.com/26345101/62276091-93fab380-b443-11e9-9e20-d93fe44c00da.png)
